src/ngraph/runtime/cpu/op/quantize.cpp

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+* Copyright 2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "ngraph/runtime/cpu/op/quantize.hpp"
+
+ngraph::op::Quantize::Quantize(std::shared_ptr<Node> input,
+                               std::shared_ptr<Node> min,
+                               std::shared_ptr<Node> max,
+                               const element::Type& type)
+    : RequiresTensorViewArgs("Quantize", {input, min, max})
+    , m_element_type(type)
+{
+    add_output(type, input->get_shape());
+    add_output(input->get_element_type(), Shape{1});
```

Comment:
This is a scalar, right? Should be `Shape{}`.
Also, `set_output_type(1, get_input_element_type(0), Shape{})` will save one of us some conflict resolution.

Reply:
I dont see anything like 'set_output_type' in ngraph on my branch. Is it recently added? Or do you mean I should create one?

Reply:
I guess it's just in my PR then, so I'll deal with it since you'll be merged long before me.